scripts/memrem.coffee

```diff
@@ -57,7 +57,7 @@ onSet = (msg) ->
 
 module.exports = (robot) ->
   robot.respond /(mem|get) (?!safesearch|list)([a-zA-Z0-9_-]*)$/i, onGet
-  robot.respond /(rem|set) (?!safesearch|list)([a-zA-Z0-9_-]*) (.*)/i, onSet
+  robot.respond /(rem|set) (?!safesearch|list)([a-zA-Z0-9_-]*) ((?:.|[\r\n])+)/i, onSet
```

Comment:
Can you add a comment for the second capture here? What does the first half do? `?:.`